feat: add evictAfterOOMThreshold per vpa (#8692)
kubernetes/autoscaler: vertical-pod-autoscaler/docs/flags.md

| `add-dir-header` |  |  | If true, adds the file directory to the header of the log messages |
| `address` | string |  ":8943" | The address to expose Prometheus metrics.  |
| `alsologtostderr` |  |  | log to standard error as well as files (no effect when -logtostderr=true) |

> | `evict-after-oom-threshold` |  |  10m0s | duration                              The default duration to evict pod that has OOMed in less than evict-after-oom-threshold since start.  |

Reviewer: nit
```suggestion
| `evict-after-oom-threshold` |  |  10m0s | duration                              The default duration to evict pods that have OOMed in less than evict-after-oom-threshold since start.  |
```
